include/vulkan/utility/vk_struct_helper.hpp

```diff
@@ -1852,7 +1852,7 @@ class InitStructHelper {
     void* p_next = nullptr;
   public:
     InitStructHelper() = default;
-    InitStructHelper(void *p_next) : p_next(p_next) {};
+    InitStructHelper(void *p_next) : p_next(p_next) {}
```

Comment:
Actually you'll need to modify the codegen. Otherwise this will just end up happening again.

See struct_helper_generator.py for modifications

Reply:
It's on my list to make a CI check to catch this https://github.com/KhronosGroup/Vulkan-Utility-Libraries/issues/75

Reply:
Done, thanks.